theano/sandbox/multinomial.py

```diff
@@ -626,3 +626,6 @@ def local_gpu_multinomial(node):
             # The dimshuffle is on the cpu, but will be moved to the
             # gpu by an opt.
             return [gpu_from_host(ret)]
+
+
+MultinomialWOReplacementFromUniform = ChoiceFromUniform
```

Comment:
I think we should make this raise a warning to help people know about the deprecation. I think you can follow this example, but change the method to just print a warning:

```
+# This is for compatibility with pickled things.  It should go away at
+# some point.
+class DownsampleFactorMaxGrad(object):
+    def __new__(self, ds, ignore_border, st=None, padding=(0, 0), mode='max'):
+        if mode == 'max':
+                return MaxPoolGrad(ds=ds, ignore_border=ignore_border, st=st,
+                                   padding=padding, mode='max')
+        else:
+            return AveragePoolGrad(ds=ds, ignore_border=ignore_border, st=st,
+                                   padding=padding, mode=mode)
```